### ISSUE TITLE
Вот такая история...

### ISSUE THREAD

**Issue by Rocky54** [Maxx53/SCMBot#519]
Думал я с ноября перейти на динамический ip так как баны заипали. А только потом когда перекинул деньги, позвонил им, узнал что у провайдера нету динамического IP, а только статические. А бобла туда им закинул на 2 месяца вперёд, так что менять  провайдера сейчас уже глупо. И теперь вот такая ситуация..... Хотелось бы узнать можно ли как то обходить баны или менять ip на статическом? :(

**Comment by nikita971** [comment]
на динамике то с трудом можно поменять, а ты про статику:/ Баны ты никак не обойдешь в любом случае, ип можно сменить, ты все эти способы знаешь.

**Comment by Rocky54** [comment]
@nikita971 

Не понял тебя)

На динамике меняется обычной перезагрузок

А по поводу способов - не знаю. На статике никак вроде

**Comment by rnd0m** [comment]
а как же твоя убер-инструкция "как сменить статитический ип"? https://github.com/Maxx53/SCMBot/issues/501
не помогает? может тогда не стоит писать инструкции хотя бы на темы, которые ты даже проверить не в состоянии, не то что разобраться? )

**Comment by Exstasy** [comment]
Лучше бы дедики купил самые дешевые и сделал бы себе мини впн ( прокси ). хотя это довольно дороговато. А впн уже щас все в бане, куда не зайти на любой айпи стим в бане. Люди не дураки ...

**Comment by Rocky54** [comment]
@rnd0m 

Да, согласен. был не прав, но я её не сам составлял :) В инете увидел, отправил сюда, так как был не за компом

**Comment by Rocky54** [comment]
@Exstasy 

То есть те кто сейчас меняют IP (если он у них динамический) попадают постоянно на IP уже с баном?

**Comment by fallenQQ** [comment]
@Rocky54  нет . он говорит про прокси , ибо  со статическим  айпишкой гг , а с динамо запросто , только вот  бот стал медленнее покупать

**Comment by Exstasy** [comment]
@fallenQQ А все потому что хотели как лучше а получили как всегда "VOLVO" Раньше задержка 0 была у всех. потом мини баны сделали на 5 мин. щас на 6-10 часов. И теперь вся школа STH пользуется анонимайзерами прокси впнами и обходит баны. Не только обходит а сканирует на задержке 0 как раньше покупая быстрее всех. Дедики покупают 30-50 предметов за 24 часа. Пора удалять ботнет. Мини сервер собрать с поддержкой 30-50 впс. И впн поставить на роутер и все. Доход вернется тот же что и был. Но вложить 300-500к проблематично... Да и гемороя полно* электричество самый трешь будет свет выкл во всем этаже* Но есть решение генератор купить на топливе и поставить на крышу ( или солнечные батареи )... Для маленьких ботоводов не че не могу сказать, мутите прокси впны фармите. для ср тоже самое. Ну а для крупных с армией ботов. Вы так же думаете как и я. Сейчас хуй просышь че будет через неделю. Опасно вкладывать много бабла. Мб на новый год уберут эти запросы обосцаные или снизят период бана. Полная муть уходим в отпуск на месяц...

**Comment by Exstasy** [comment]
@fallenQQ А все потому что хотели как лучше а получили как всегда "VOLVO" Раньше задержка 0 была у всех. потом мини баны сделали на 5 мин. щас на 6-10 часов. И теперь вся школа STH пользуется анонимайзерами прокси впнами и обходит баны. Не только обходит а сканирует на задержке 0 как раньше покупая быстрее всех. Дедики покупают 30-50 предметов за 24 часа. Пора удалять ботнет. Мини сервер собрать с поддержкой 30-50 впс. И впн поставить на роутер и все. Доход вернется тот же что и был. Но вложить 300-500к проблематично... Да и гемороя полно* электричество самый трешь будет свет выкл во всем этаже* Но есть решение генератор купить на топливе и поставить на крышу ( или солнечные батареи )... Для маленьких ботоводов не че не могу сказать, мутите прокси впны фармите. для ср тоже самое. Ну а для крупных с армией ботов. Вы так же думаете как и я. Сейчас хуй просышь че будет через неделю. Опасно вкладывать много бабла. Мб на новый год уберут эти запросы обосцаные или снизят период бана. Полная муть уходим в отпуск на месяц...

**Comment by fallenQQ** [comment]
@Exstasy   габен нечего не будет менять, хотя  не понимаю его логики, бабло он то все равно рубит

**Comment by kyotooo** [comment]
Логика в ддосе.
Когда 1000человек отправляет в час тысячи запросов. Их Сервакам не слишком збс. Это вам кажется, что среди сотен тысяч - вы никакой нагрузки не создаете - всё очевидно.

**Comment by Dampir93** [comment]
@Exstasy Привет) Научи менять айпишник. Я вижу ты в теме.

**Comment by nikita971** [comment]
Давайте всех всему научим, потом будем удивляться почему все vps перебанены ;)

**Comment by nikita971** [comment]
Тут логика такая, если хочешь иметь деньги -сам всему научишься

**Comment by Exstasy** [comment]
@nikita971 Да тут не учиться надо а нанять шарящего программиста и пусть ищет уязвимые места и хуярит бота. Макс в отпуске на долгосрочное время) Я не че не хочу говорить по поводу Макса он написал прогу она работала, стим ввели огр он тут не причем и ничего не обязан делать=) Кто знает что делать уже делает. Щас куда не зайди опять ботоводы везде))) И всем все равно на баны и прочий бред. Умеют находить выход из сложной ситуации. Я вот только не понимаю одного все ботоводы сидят в кс го. А про доту что забыли? Мозгами поразмыслите мб откроете золотое дно для себя) Я щас лично тестю доту 2 неделю. Закупаешься всякой херней по дешевке через неделю в пятницу - суб все продаю. изи 30-40% от капитала приходит. Если так подумать то это куда выгоднее чем каждый день сидеть фармить целый день менять айпи и так далее... 3-5 лотов ставишь на ср задержке и работает все на изи... Так что думайте сами. А в кс щас наваришь 150руб за день и все. Куда лучше 100к в оборот пустить и иметь 30-50к чистыми в неделю.  Да даже в 2 недели... все потом вывести с потерей 20% и получится циферка выше ср/ заработной платы города... Решайте сами чего вы хотите добиться) Тупо на обороте можно поднимать столько денег сколько зам директора не получают на работе. И это никогда не закроют вот в чем фишка.
